content/terraform-modules/root/codefresh-pipeline.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "CodeFresh Pipeline for Root Modules"
+description: "Example of the pipeline we use to build the `terraform-root-modules` docker image"
+---
+
+{{% include-github org="cloudposse" repo="terraform-root-modules" ref="master" file="codefresh.yml" title="Example codefresh.yml" language="yaml" type="code-block" %}}
```

Comment:
Let's replace `type="code-block"` with just `inline=false` or `block=true`

Reply:
at first, i had `code-block=true`, but replaced it with `type` so that it can be extended to support different formatting options. The `dialog` shortcode accepts `type`.  I don't like `inline` because it does not convey markdown. Other type could be `markdown`.